src/containers/access/src/sagas/saveNewFormTemplate.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+import isPlainObject from 'lodash/isPlainObject';
+import {
+  call,
+  put,
+  select,
+  takeEvery,
+} from '@redux-saga/core/effects';
+import {
+  DataApiActions,
+  DataApiSagas,
+} from 'lattice-sagas';
+import { Logger } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type { WorkerResponse } from 'lattice-sagas';
+import type { SequenceAction } from 'redux-reqseq';
+
+import getESIDFromConfig from '../../../../utils/getESIDFromConfig';
+import { AppTypes, PropertyTypes } from '../../../../core/edm/constants';
+import { selectPropertyTypeIDsByFQN } from '../../../../core/redux/selectors';
+import { ERR_ACTION_VALUE_TYPE } from '../../../../utils/Errors';
+import { APP_PATHS } from '../../../app';
+import {
+  SAVE_NEW_FORM_TEMPLATE,
+  saveNewFormTemplate,
+} from '../actions';
+
+const { createOrMergeEntityData } = DataApiActions;
+const { createOrMergeEntityDataWorker } = DataApiSagas;
+
+const { FORM } = AppTypes;
+const {
+  RJSF_JSON_SCHEMA,
+  RJSF_UI_SCHEMA,
+  TYPE,
+} = PropertyTypes;
+
+const LOG = new Logger('saveNewFormTemplateSagas');
+
+function* saveNewFormTemplateWorker(action :SequenceAction) :Saga<WorkerResponse> {
```

Comment:
I don't think this belongs in the "source" since this is only being used in the story. As it is, this is going to be included in the bundle.

Reply:
How would you propose to handle it otherwise? It's really not much code.

Reply:
I would create a second `Sagas.js` file that is solely used for stories and would look into explicitly excluding `stories/**` during the build.

You're right, though. At the moment, it's not much code, it's not going to make a material difference.